### DISCUSSION TITLE
How use videoRef for take the current time of a video ?

### DISCUSSION THREAD

**Discussion by bpernot** [ValentinH/react-easy-crop#413] [Q&A]
![image](https://user-images.githubusercontent.com/55028058/195399360-9780e5fb-9be8-49aa-aacf-a67930dbddac.png)

I do this and i want to when i play the video i got the current time like onClick => play => and time go 1, 2 ,3s etc 
I don't found the solution to do that if anyone have a idea ?

Thanks all !

**Comment by ValentinH** [comment]
I'm sorry but I don't really understand what you are trying to do. Also, how is this related to this library? Do you manage to do what you want with a native video instead of using `react-easy-crop` ?
